snc.sh

```diff
@@ -169,6 +169,31 @@ export OPENSHIFT_INSTALL_INVOKER="codeReadyContainers"
 
 ${OPENSHIFT_INSTALL} --dir $INSTALL_DIR create cluster --log-level debug
 
+# export the kubeconfig
+export KUBECONFIG=$INSTALL_DIR/auth/kubeconfig
+
+# Add kubelet-bootstrap-cred-manager daemonset
+oc apply -f kubelet-bootstrap-cred-manager-ds.yaml
+
+# Delete the current csr signer to get new request. 
+oc delete secrets/csr-signer-signer secrets/csr-signer -n openshift-kube-controller-manager-operator
+
+# Wait for 5 min to make sure cluster is stable again.
+sleep 300
```

Comment:
do you think `oc wait --for=condition=Ready pod/xxx` could be used here rather than sleep?

Reply:
@cfergeau no, wait is good for a single pod which we know about, here we don't know the which all the pods are going to terminated and come back again so need to put a sleep.

Reply:
Checking that a pod which starts very late is available might be a good addition to the sleep so that we are sure we waited long enough?

Reply:
@cfergeau might be, but after the 2-3 mins wait we already check with `openshift-install --wait-for ..` which is more accurate then we are trying to go with different pods and it schedule.